proposed/http-middleware/middleware.md

```diff
@@ -73,34 +72,34 @@ interface MiddlewareInterface
 {
     /**
      * Process an incoming server request and return a response, optionally delegating
-     * to the next middleware component to create the response.
+     * response creation to a handler.
      *
      * @param ServerRequestInterface $request
-     * @param DelegateInterface $delegate
+     * @param HandlerInterface $handler
      *
      * @return ResponseInterface
      */
     public function process(
         ServerRequestInterface $request,
-        DelegateInterface $delegate
+        HandlerInterface $handler
```

Comment:
Right I think I see things a bit differently now: the middleware decorates a handler. Whether there are other middlewares involved is a detail.

👍  for me then thanks